providers/tests/google/cloud/operators/test_datastore_system.py

```diff
@@ -44,8 +44,8 @@ def teardown_method(self):
 
     @provide_gcp_context(GCP_DATASTORE_KEY)
     def test_run_example_dag(self):
-        self.run_dag("example_gcp_datastore", CLOUD_DAG_FOLDER)
+        self.run_dag("example_gcp_datastore", CLOUD_DAG_FOLDER)  # this dag does not exist?
 
     @provide_gcp_context(GCP_DATASTORE_KEY)
     def test_run_example_dag_operations(self):
-        self.run_dag("example_gcp_datastore_operations", CLOUD_DAG_FOLDER)
+        self.run_dag("example_gcp_datastore_operations", CLOUD_DAG_FOLDER)  # this dag does not exist?
```

Comment:
How on earth have these tests been passing? I guess they haven't as we don't regularly run these system tests? 

These dags were removed/moved/renamed in https://github.com/apache/airflow/pull/23108 by @bhirsz.

I guess the google system tests can't have been fully run since 2022 then. Yay

Reply:
i don't think we run them. we just spend a lot of time updating them in this PR :)